{
  "name": "oslllo-validator-exception",
  "version": "2.0.1",
  "description": "All in one Javascript validator but throws an exception on false",
  "main": "src/index.js",
  "scripts": {
    "test": "npm run build && nyc mocha && npm run lint",
    "eslint": "node_modules/.bin/eslint",
    "lint": "npm run eslint -- src test",
    "webpack": "node_modules/.bin/webpack",
    "build": "npm run browser:build",
    "browser:build": "npm run browser:build.dev",
    "browser:build.dev": "npm run webpack -- --config webpack.dev.js",
    "coverage": "node_modules/.bin/nyc report --reporter=lcovonly"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/oslllo/validator-exception.git"
  },
  "author": "",
  "license": "MIT",
  "files": [
    "src",
    "LICENCE",
    "README.md"
  ],
  "bugs": {
    "url": "https://github.com/oslllo/validator-exception/issues"
  },
  "homepage": "https://docs.oslllo.com/validator/master",
  "dependencies": {
    "oslllo-validator": "^2.0.1"
  },
  "keywords": [
    "oslllo",
    "validator",
    "validator-exception",
    "exception",
    "javascript",
    "all-in-one-validator",
    "all-in-one"
  ],
  "devDependencies": {
    "@babel/cli": "^7.15.4",
    "@babel/core": "^7.15.5",
    "@babel/preset-env": "^7.15.4",
    "chai": "^4.3.4",
    "clean-webpack-plugin": "^3.0.0",
    "coveralls": "^3.1.1",
    "eslint": "^7.32.0",
    "jsdom": "^16.7.0",
    "mocha": "^8.4.0",
    "mocha-lcov-reporter": "^1.3.0",
    "nyc": "^15.1.0",
    "webpack": "^5.52.0",
    "webpack-cli": "^4.8.0",
    "webpack-merge": "^5.8.0"
  },
  "engines": {
    "node": ">= 10.0"
  }
}
